### PR TITLE
finished the tree intersection code challenege

### PR DESCRIPTION
**Pull request by wosunkwo**

### CODE REVIEW

**Comment by wosunkwo** [comment]
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

File: code401-challenges/src/main/java/code401/challenges/treeIntersection/TreeIntersection.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                dummyCnt++;
+                if(duplicateHash.get(node.getValue()) == null){
+                    duplicateHash.put(node.getValue(), dummyCnt);
+                }
+                hashFirstTree(node.getLeft(), duplicateHash, dummyCnt);
+                hashFirstTree(node.getRight(), duplicateHash, dummyCnt);
+            }
+    }
+
+    private void hashSecondTree(Node<Integer> node, HashMap<Integer, Integer> duplicateHash, int dummyCnt, ArrayList<Integer> duplicateArrList){
```
Comment (evanslaton) [inline code401-challenges/src/main/java/code401/challenges/treeIntersection/TreeIntersection.java:38]: This method, since it is doing the same thing as the previous method just on a different tree, make your code not DRY

Comment (wosunkwo) [reply code401-challenges/src/main/java/code401/challenges/treeIntersection/TreeIntersection.java:38]: It is doing a different thing than the previous method
